content/influxdb/cloud/organizations/bucket-schema/_index.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Manage bucket schemas
+seotitle: Manage bucket schemas in InfluxDB
+description: Manage bucket schemas in InfluxDB using the influx CLI.
+menu:
+  influxdb_cloud:
+    name: Manage bucket schemas
+    parent: Manage organizations
+weight: 105
+influxdb/cloud/tags: [buckets, bucket-schema, schema]
+---
+
+Use *bucket schemas* to restrict a bucket's columns and data types.
+
+A **bucket schema** enforces the shape of data in a bucket.
+By default, buckets have an `implicit` **schema-type**  which conforms
+to data as it is written to InfluxDB.
+
+Buckets with the `explicit` schema-type allow you to assign your own named schemas
+and reject write requests that do not conform to a specific schema.
+
+### Create a bucket schema
+Use the `influx` CLI to set the schema-type and one or more schemas for your bucket:
+
+1. Create a measurement schema *columns file* using CSV, JSON, or [Newline delimited JSON (NDJSON)](http://ndjson.org/). Define the name, type, and data type of each column.
```

Comment:
Good idea. In this context, they're effectively the same. bucket-schema is essentially a join of a measurement schema to a bucket. I was trying to be careful in case measurement schema is used in other contexts (now or in the future).

Reply:
...also, "measurement name" is used to distinguish schema.